Devscord.DiscordFramework/Framework/Commands/Responses/Resources/Responses.resx

```diff
@@ -261,4 +261,10 @@
   <data name="TryToGoogleIt" xml:space="preserve">
     <value>Spróbuj to wygooglować :wink:\nMasz tu link: {{link}}</value>
   </data>
+  <data name="ComplaintsChannelAlreadyExists" xml:space="preserve">
+    <value>Kanał na skargi już istnieje na tym serwerze.</value>
+  </data>
+  <data name="ComplaintsChannelHasBeenCreated" xml:space="preserve">
+    <value>Kanał na skargi został utworzony pomyślnie.</value>
+  </data>
```

Comment:
czym to się różni?

Reply:
w sensie?

Reply:
z tego co ja tu widzę to oba komunikaty są chyba w tej samej sytuacji wyświetlane. Bo skoro "Kanał na skargi został utworzony pomyślnie" to oznacza, że "Kanał na skargi już istnieje na tym serwerze". Dla mnie oba komunikaty są jak synonimy i nie wiem w jakich sytuacjach jest dany komunikat, czy może oba są do tej samej sytuacji?

Reply:
no nie, "Kanał na skargi już istnieje na tym serwerze." jest w momencie kiedy ktoś wywołał tą komendę a kanał już istnieje